rmic/src/main/java/org/glassfish/rmic/tools/asm/Instruction.java

```diff
@@ -610,14 +610,14 @@ int size(ConstantPool tab) {
           case opc_tableswitch: {
             SwitchData sw = (SwitchData)value;
             int n = 1;
-            for(; ((pc + n) % 4) != 0 ; n++);
+            for(; ((pc + n) % 4) < 0 ; n++);
```

Comment:
:eyes:

Reply:
```suggestion
            for(; ((pc + n) % 4) < 0 ; n++);
``` 
I flipped the coin. The button said I am right giving me a right benefit of a doubt it changes something in our metaverse. 

I will look more into the detail of the scan, to maybe give more less 'too specific' issues to the board - not every shot is a headshot like they say :) Let's try again. Thanks!